src/pure.ts

```diff
@@ -16,9 +16,11 @@ export type {
   RenderResult as RenderAPI,
 } from './render';
 export type { RenderHookOptions, RenderHookResult } from './renderHook';
+export type { Config } from './config';
 
 export { act };
 export { cleanup };
+export { configure, resetToDefaults } from './config';
```

Comment:
Is `resetToDefaults` exported from DTL? Couldn't find it on their docs

Reply:
No it's not, only `configure` is, but I think it's good to have that for testinging convenience. Wdyt?

Reply:
Re `defaultHidden`: we first need to have https://github.com/callstack/react-native-testing-library/pull/1064 merged as I belive that `defaultHidden` is the default value of `hidden` query option, which we are yet to implement.